packages/components/src/ControlGroup/ControlGroup.js

```diff
@@ -40,4 +44,26 @@ function ControlGroup(props, forwardedRef) {
 	);
 }
 
-export default contextConnect(ControlGroup, 'ControlGroup');
+/**
+ * `ControlGroup` is a layout component that contains control elements (e.g. `TextInput` or `Select`).
+ *
+ * @example
+ * ```jsx
+ * import { Button, ControlGroup, TextInput } from `@wp-g2/components`
+ * import { ui } from `@wp-g2/styles`
+ *
+ * function Example() {
+ *   return (
+ *     <ControlGroup>
+ *       <TextInput placeholder="First name" />
+ *       <Button variant="primary" />
+ *     </ControlGroup>
+ *   );
+ * }
+ * ```
+ *
+ * @type {import('@wp-g2/create-styles').PolymorphicComponent<'div', import('./types').Props>}
+ */
+const ConnectedControlGroup = contextConnect(ControlGroup, 'ControlGroup');
```

Comment:
We put the final description and example for the component on the connected one as that's what's actually exported, that way you get it in intellisense. If we put it on the actual component it would get lost.

Reply:
Gotcha! I'm so thankful we have these initial components as example 🙏